### PR TITLE
#162949203 Integrate travis

### PR DESCRIPTION
**Pull request by MbugwaSami**
**What does this PR do?**

- Integrate Travis to the repo and add  Readme badge

**Description of the Task to be completed**
- Add travis.yml file
- Integrate ah-unsullied repo to Travis account
- Add Travis badge to Readme

**How should this be manually tested?**
- Navigate to the [Chore branch](https://github.com/andela/ah-the-unsullied/tree/ch-integrate-travis-162949203) to view the badge
- Navigate to [ah-unsullied on Travis](https://travis-ci.org/andela/ah-the-unsullied) to view the builds

**What are the relevant pivotal tracker stories?**

- #162949203

**Screenshots (if appropriate)**

- N/A

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+
+# Install dependencies
+install:
+  - pip install -r requirements.txt
+  - pip install pytest
+  - pip install coveralls
+
+before_script:
+
```
Comment (Allan690) [inline .travis.yml:17]: You need to create a database for the migrations, before running the script(before script)

Comment (MbugwaSami) [reply .travis.yml:17]: Let me add that.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+# Run tests
+script:
+  - coverage run --source=authors/ manage.py test 
+  - coverage report
+
+# Generate coverage report
+after_success:
+  - coveralls
```
Comment (kelvinwm) [inline .travis.yml]: check on pep 8 style guide. At the end of the code, the last line should be blank

Comment (MbugwaSami) [reply .travis.yml]: on it.

**Review by nesh-dev — CHANGES_REQUESTED** [review]
The build should fail with an indication of no tests. It seems the build exits with an error.

**Review by kelvinwm — DISMISSED** [review]

**Review by Allan690 — DISMISSED** [review]

**Review by nesh-dev — DISMISSED** [review]

**Review by kelvinwm — DISMISSED** [review]

**Review by Allan690 — DISMISSED** [review]

**Review by nesh-dev — DISMISSED** [review]

**Review by kwanj-k — DISMISSED** [review]

**Review by indungu — APPROVED** [review]
LGTM

**Review by kwanj-k — APPROVED** [review]

**Pull request merged**